src/command_modules/azure-cli-role/tests/test_graph.py

```diff
@@ -85,3 +85,76 @@ def body(self):
         # delete app
         self.cmd('ad app delete --id {}'.format(app_id_uri))
         self.cmd('ad app list --identifier-uri {}'.format(app_id_uri), checks=NoneCheck())
+
+
+class GraphGroupScenarioTest(VCRTestBase):
```

Comment:
Done. 
I recommend we book a sprint and convert all to the new style. Inconsistencies within a file is not ideal for code maintenance